packages/build/src/npm-packages.ts

```diff
@@ -10,14 +10,6 @@ export function bumpNpmPackages(version: string): void {
     return;
   }
 
-  execFileSync('git', ['--version'], { stdio: 'inherit' });
```

Comment:
Note: I had this in directly on master for testing as it took me a while to understand the problem...